MessengerOnFileForLinux/UserService/test/SignInTest.cpp

```diff
@@ -1,4 +1,4 @@
-/*#include <gtest/gtest.h>
+#include <gtest/gtest.h>
 
 #include <SignInTestCore.hpp>
 
```

Comment:
24 linijka:
przecinek ,user

Reply:
Wgl nie rozumiem czemu w testach tych wołamy funkcje z FileHandlingu -> rozumiem jeszcze to z get, ale to z remove? Trochę dziwne

Reply:
done.
W setUpie robimy rejestre. Remove służy tu jako wyrejestrowanie użytkownika

Reply:
Czy nie powinniśmy w SetUp konfigurować folder od zera, a TearDown go usuwać? Tak by testy były niezależne w 100%, a nie połowe sprzątać w testach.

Reply:
Tak jest, ale to jest szczególny przypadek bo chcemy w trakcie testu zasymulować sobie że użytkownik jest niezarejestrowany